clicommand/agent_start.go

```diff
@@ -798,8 +801,9 @@ func handlePoolSignals(l logger.Logger, pool *agent.AgentPool) chan os.Signal {
 // shutdownHook looks for a shutdown hook script in the hooks path and executes it
 // if it's available. This is very similar to the bootstrap hook code. Just run at
 // the agent level.
-func shutdownHook(log logger.Logger, hooksPath string) {
-	p, err := hook.Find(hooksPath, "shutdown")
+func shutdownHook(log logger.Logger, cfg AgentStartConfig) {
```

Comment:
We now need `cfg.NoColor` in addition to `cfg.HooksPath`, so receive the entire `cfg AgentStartConfig` instead of just `hooksPath string`